js/project-list.js

```diff
@@ -0,0 +1,58 @@
+/* Project Card */
```

Comment:
Let's change this and other "card" occurrences in this file to "project list item" 👍